File: 251009/친근하지 않은 수/unfriendly-number.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;

    int count = 0;
    for(int i = 0; i <= a; i++) {
        if(i % 2 != 0 && i % 3 != 0 && i % 5 != 0) {
            count++;
        }
    }
    cout << count;
    return 0;
}

File: 251030/친근하지 않은 수/unfriendly-number.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = 0;
        int count = 0;
        a = int.Parse(Console.ReadLine());
        for(int i = 1; i <= a; i++) {
            if(i % 2 != 0 && i % 3 != 0 && i % 5 != 0) {
                count++;
            }
        }
        Console.Write(count);
    }
}
